### PR TITLE
Add 2 testbenches for computational requirements when running `viash …

### PR DESCRIPTION
**Pull request by Grifs**
…run` or `viash test`

Please review testbench check values. The behaviour seems inconsistent to me.

### CODE REVIEW

File: src/test/resources/testbash/auxiliary_computational_requirements/check_computational_requirements.vsh.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+functionality:
+  name: auxiliary_computational_requirements
+  version: 0.1
+  description: |
+    Check setting of computational requirements
+  resources:
+    - type: bash_script
+      text: |
+        if [ -n "$VIASH_META_CPUS" ]; then
+          echo "cpus: $VIASH_META_CPUS"
```
Comment (rcannood) [inline src/test/resources/testbash/auxiliary_computational_requirements/check_computational_requirements.vsh.yaml:10]: could you use `$meta_cpus` instead? `$VIASH_META_CPUS` is not meant to be used by end users

File: src/test/resources/testbash/auxiliary_computational_requirements/check_computational_requirements.vsh.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        else
+          echo "cpus unset"
+        fi
+
+        if [ -n "$VIASH_META_MEMORY" ]; then
+          echo "memory: $VIASH_META_MEMORY"
```
Comment (rcannood) [inline src/test/resources/testbash/auxiliary_computational_requirements/check_computational_requirements.vsh.yaml:16]: Please use something like`$meta_memory_mb` or `$meta_memory_gb`

**Review by rcannood — APPROVED** [review]

**Review by rcannood — CHANGES_REQUESTED** [review]
Eep, clicked wrong button. Changes requested!

File: src/test/resources/testbash/auxiliary_computational_requirements/check_computational_requirements.vsh.yaml (hunk 1 above)
Comment (Grifs) [reply src/test/resources/testbash/auxiliary_computational_requirements/check_computational_requirements.vsh.yaml]: done!

File: src/test/resources/testbash/auxiliary_computational_requirements/check_computational_requirements.vsh.yaml (hunk 2 above)
Comment (Grifs) [reply src/test/resources/testbash/auxiliary_computational_requirements/check_computational_requirements.vsh.yaml]: done!

**Pull request merged**